discovery-provider/default_config.ini

```diff
@@ -24,6 +24,7 @@ session_cookie_secure = false
 [web3]
 host = localhost
 port = 8545
+provider_url = 
```

Comment:
why default this to empty instead of `provider_url = http://audius_ganache_cli:8546` (double check this)

Reply:
Service providers are still using `host` and `port` for their URLs.

Changing this would result in a breaking change, since the code prioritizes `provider_url` over `host` and `port`